README.adoc

```diff
@@ -1,35 +1,86 @@
-// Before publishing your new repository:
-// 1. Write the readme file
-// 2. Update the issues link in Contributing section in the readme file
-// 3. Update the discussion link in config.yml file in .github/ISSUE_TEMPLATE directory
+= Java Operating System Statistics (jos_01)
 
-= repo-template
-
-// Add a short description of your project. Tell what your project does and what it's used for.
-
-This is a template repository for Teragrep organization.
+This project will extract statistics from the operating system and running processes into Java objects.
+You can use this to easily get information such as memory usage about running processes and the operating system itself.
+The information provided is based on the Linux filesystem's /proc files.
 
 == Features
 
-// List your project's features
+Check if a process is alive using process id.
+Get statistics on the OS or any running process using your system's /proc files as the source of truth.
+High level functions calculate process performance statistics automatically, such as residentSetSize()
+Create objects representing the current state of a /proc file and get access to the file's values.
+
+This project can be used to easily create system or process monitoring solutions in Java applications.
 
 == Documentation
 
 See the official documentation on https://docs.teragrep.com[docs.teragrep.com].
 
 == Limitations
 
-// If your project has limitations, please list them. Otherwise remove this section.
+Only supports Linux-based operating systems
+Relies on availability of /proc files, so JOS-01 needs to have access to these files.
 
 == How to [compile/use/implement]
```

Comment:
remove brackets and use one of following titles:

- How to Compile
- How to Use
- How to Implement